Project/frontend/src/App.js

```diff
@@ -1,9 +1,57 @@
 import { Chessboard } from "react-chessboard";
+import { useState } from "react";
 
 export default function App() {
+  const [position, setPosition] = useState('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
+  function clearBoard() {
+    setPosition({});
+  }
+  function startPosition() {
+    setPosition('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
+  }
+  function updatePosition(updatedposition) {
+    setPosition(updatedposition);
+    console.log(updatedposition);
+  }
+  
+  let bturn = {"turn":"black",position};
+  let wturn = {"turn":"white",position};
+  function submitPositionB() {
+   fetch('http://localhost:3000/', {
```

Comment:
Add the `await` keyword before this (so like `await fetch`). This will make it so that the program waits for the response to finish before updating